src/codegen/function_body_chunk.rs

```diff
@@ -244,12 +244,13 @@ impl Builder {
                     &bounds,
                     &bounds_names,
                     false,
+                    &self.callbacks[callback_pos].bound_name,
```

Comment:
Why not `&trampoline.bound_name,` in this for?

Reply:
I need to check if it's the same. Thanks for the lead!

Reply:
I confirm, it's the same!

Reply:
Actually only for trampoline. Destroy callbacks have this issue. We need to update their bounds.